packages/flutter_test/lib/src/goldens.dart

```diff
@@ -77,13 +77,13 @@ abstract class GoldenFileComparator {
     if (identical(test, master))
       return ComparisonResult(passed: true);
 
-    if (test == null
-      || master == null
-      || test.isEmpty
-      || master.isEmpty) {
+    if (test == null ||
+      master == null ||
```

Comment:
These still need to be indented to align under `test ==` (or, probably better, be all on one line).

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#align-expressions